framework/core/js/src/common/index.ts

```diff
@@ -11,9 +11,11 @@ import 'jquery.hotkeys/jquery.hotkeys';
 
 import relativeTime from 'dayjs/plugin/relativeTime';
 import localizedFormat from 'dayjs/plugin/localizedFormat';
+import dayjsPlugins from './utils/dayjsPlugins';
```

Comment:
avoid to use `XXXXXPlugins`, considering another more specific name like `humanTimeFormatter`.

Reply:
It's not only for `humanTime`.

Reply:
so i said `like`, better not to use common name